# Text Compare
# Left: /l/left.c (UTF-8, lf)
# Right: /r/right.c (UTF-8, cr_lf)
# Mode: display: context (3 lines); ignore unimportant: no
# Rules: format: Everything Else; algorithm: standard (auto); skew: 300; closeness: 500‰; never align: no; unimportant: none; line endings: ignored; orphan lines: always important
#
--- /l/left.c	2026-10-19T13:12:00Z
+++ /r/right.c	2026-10-19T14:00:00Z
@@ -1,2 +1,2 @@ unimportant
-x
-y
+x
+y
